### ISSUE TITLE
On failed transection give success 

### ISSUE THREAD

**Issue by deepsinghrathi** [PatilShreyas/EasyUpiPayment-Android#151]
When I dial wrong pin or payment goes failed then your system give success. I try many time please you try also.

**Comment by deepsinghrathi** [comment]
Sir my app is live application, show please answer early.

**Comment by dynomus** [comment]
@deep191717 what do you pass as txnId and merchant code ?

**Comment by marutichintan** [comment]
My Suggest not to use this plugin, because you cannot verify the Transaction with Bank whether it success or not. if you want to make sure, Install SMS Reader App in you mobile and Confirm using Bank Messages.

if you want Market Solution try https://upigateway.com

**Comment by bkubes** [comment]
Yes, i too encountered the same issue, i wrongly entered the pin number but still it is showing the SUCESS transaction, shreyas, could you look into this?

**Comment by rudrakshya** [comment]
Is it return UTR no?

**Comment by Vishalvp3618454** [comment]
> When I dial wrong pin or payment goes failed then your system give success. I try many time please you try also.

I have the same issue.

**Comment by Vishalvp3618454** [comment]
https://stackoverflow.com/questions/75800423/android-upi-intent-phonepe-returns-status-as-success-instead-of-failure-when

I have try to add Configure Template. but not successfully working. Any have a solution?
